000-index.md

```diff
@@ -18,5 +18,6 @@ weight: 001
     - [Credential Sets](802-credential-sets.md)
     - [Repositories](803-repositories.md)
     - [Well known custom actions](804-well-known-custom-actions.md)
+    - [Disconnected Scnenarios](805-airgap.md)
```

Comment:
Looks like an extra `n` snuck in to `Scenarios`.

Reply:
Fixed.